device-data/types/basal/suppressed.md

```diff
@@ -1,5 +1,279 @@
 <!-- non-generated document! all areas editable -->
 
-Coming soon!
+## the `suppressed` field on `temp`s and `suspend`s
 
-<!-- TODO: document how `suppressed` works when a `temp` or `suspend` basal crosses > 1 segment of a basal schedule -->
\ No newline at end of file
+Some insulin pump data protocols provide enough information for us to track various aspects (e.g., `rate`, `scheduleName`) of the basal that *would have been in effect* had the currently active `temp` or `suspend` basal **not** been programmed (or triggered). Where this information is available, we provide it in as much detail as possible as an embedded object in the `suppressed` field on a `temp` or `suspend` basal interval.
+
+In general, this `suppressed` object need and **must** only contain the bare minimum of information:
+- the `type` (= `basal`)
+- the `deliveryType` (only `scheduled` if the currently active basal is a `temp`; otherwise (i.e., if the currently active basal is a `suspend`), may be `scheduled` *or* `temp`)
```

Comment:
There is production data that has temp basal suppressing temp basal suppressing temp basal suppressing temp basal suppressing temp basal suppressing scheduled basal. So, deliveryType can be temp if the currently active basal is temp, as well. For an example, see: https://gist.github.com/darinkrauss/7a91115d0d3475e4b11e559bf6dcdcaf